convert2rhel/backup.py

```diff
@@ -189,33 +189,55 @@ def pop(self):
 
         return restorable
 
-    def pop_all(self):
+    def pop_all(self, _honor_partitions=False):
         """
         Restores all RestorableChanges known to the Controller and then returns them.
 
         :returns: List of RestorableChange objects that were known to the Controller.
         :raises IndexError: If there are no RestorableChanges currently known to the Controller.
 
         After running, the Controller object will not know about any RestorableChanges.
-        """
-        # This code ignores partitions.  Only restore_to_partition() honors
-        # them.
-        restorables = [r for r in self._restorables if r is not self.partition]
 
-        if not restorables:
+        .. note:: _honor_partitions is part of a hack for 1.4 to let us split restoring changes into
+            two parts.  During rollback, the part before the partition is restored before the legacy
+            backups.  Then the remainder of the Changes managed by BackupController are restored.
+            This can go away once we merge all of the legacy backups into RestorableChanges managed
+            by BackupController.
```

Comment:
```suggestion
            This can go away once we merge all of the legacy backups into RestorableChanges managed
            by BackupController.
            
            .. seealso:: Jira ticket to track porting to BackupController: https://issues.redhat.com/browse/RHELC-1153
```